src/streaming/controllers/StreamController.js

```diff
@@ -1607,6 +1607,10 @@ function StreamController() {
         }
     }
 
+    function refreshManifest() {
+        manifestUpdater.refreshManifest();
```

Comment:
This should take `isUpdating` in the `ManifestUpdater` into account. If the manifest is currently being updated we dont want to trigger a another refresh.

Reply:
https://github.com/Dash-Industry-Forum/dash.js/pull/4330/commits/e3f49d3c3e1aeb4f33144d503747734886290512